src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -696,22 +696,25 @@ export class BaseEditCourseForm extends React.Component {
             courseSubmitting={submitting}
             {...this.props}
           />
+          <Link to={`/courses/${uuid}/rerun`}>
+            <button
+              id="new-run-button"
+              className="btn btn-block rounded mt-3"
+              disabled={!pristine || courseInReview}
+            >
+              <Icon className="fa fa-plus" /> Add Course Run
+            </button>
+          </Link>
           <ButtonToolbar className="mt-3">
-            <Link to={`/courses/${uuid}/rerun`}>
-              <button
-                className="btn btn-outline-primary"
-                disabled={!pristine || courseInReview}
-              >
-                Re-run Course
-              </button>
-            </Link>
             <ActionButton
               disabled={courseInReview || submitting}
               labels={{
-                default: 'Save Course',
+                default: 'Save & Continue Editing',
                 pending: 'Saving Course',
                 complete: 'Course Saved',
               }}
+              className="btn-outline-primary"
```

Comment:
I don't think you need to pass this in when you already have your `primary` indicator variable that sets this class when `primary = false`

Reply:
Good catch forgot I was still passing this.